packages/quantic/force-app/main/default/lwc/exampleSearch/resultTemplates/youtubeResultTemplate.html

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="lgc-bg slds-border_bottom">
+    <lightning-tile type="media">
+      <div class="slds-grid slds-wrap slds-m-vertical_x-small ">
+        <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_3-of-12 slds-text-align_left" style="font-weight: 400; font-size:12px">
+          <span><lightning-icon slot="media" icon-name={icon} size="small"></lightning-icon></span>
+          <span class="slds-m-left_xx-small">{label}</span>
+        </div>
+        <div class="slds-col slds-size_1-of-1 slds-large-size_6-of-12 slds-align-middle slds-large-order_2">
+          <template if:true={result.isRecommendation}><div><lightning-badge label="Recommended" class="recommended-badge" icon-name="utility:favorite"></lightning-badge></div></template>
+        </div>
+        <div class="slds-col slds-order_3 slds-large-order_3 slds-size_1-of-1 slds-large-size_3-of-12 slds-text-align_right" style="font-weight: 400; font-size:12px; color:#3E3E3C">
+          <p class="slds-truncate" title="Date">
+            <lightning-formatted-date-time value={result.raw.date}></lightning-formatted-date-time>
+          </p>
+        </div>
+      </div>
+      <div class="slds-grid slds-wrap slds-m-vertical_small ">
+        <div class="slds-col slds-order_1 slds-large-order_1 slds-size_1-of-1 slds-large-size_3-of-12 slds-text-align_left">
+            <iframe style="height:85px; width:130px;" src={videoSourceId}></iframe>
+        </div>
+        <div class="slds-col slds-order_2 slds-large-order_2 slds-size_1-of-1 slds-large-size_9-of-12 slds-text-align_left slds-p-around_none">
+          <div class="slds-grid slds-wrap">
+              <div class="slds-col slds-order_1 slds-large-order_1 slds-size_1-of-1 slds-large-size_12-of-12 slds-text-align_left" style="font-weight: 700;">
+                  <h3 style="color:black" class="slds-truncate"><c-quantic-result-link result={result} engine-id={engineId}></c-quantic-result-link></h3>
+              </div>
+              <div class="slds-col slds-order_1 slds-large-order_1 slds-size_1-of-1 slds-large-size_12-of-12 slds-text-align_left" style="font-weight: 400;">
+                <span>{result.Excerpt}</span>
+              </div>
+              <div class="slds-col slds-order_1 slds-large-order_1 slds-size_1-of-1 slds-large-size_12-of-12 slds-text-align_left" style="font-weight: 400; font-size: 10px">
+                  <span>
+                    <lightning-icon icon-name="utility:clock" size="xx-small"></lightning-icon>
+                    <span class="slds-m-left_xxx-small slds-m-top_small">5:04 min</span>
```

Comment:
`5:04 min`, get the duration from the indexed data.

Reply:
the video duration is in milliseconds. we would need a function (util function i think) for the mm:ss format. I referred how it was done in JSUI, the function is quite complex there. I am trying out if I can do it in a less complex way.